reframe/core/pipeline.py

```diff
@@ -609,6 +609,22 @@ def pipeline_hooks(cls):
     #:        }
     #:    }
     #:
+    #: Here are some examples to understand how to set the performance tuple,
+    #: in case the expected reference is a negative or a positive number:
+    #:
+    #:   ===========================  ============  ==========  ===========
+    #:   **Performance Tuple**        **Expected**  **Lowest**  **Highest**
+    #:   (100, -0.01, 0.02, 'MB/s')   100 MB/s      99 MB/s     102 MB/s
+    #:   (100, -0.01, None, 'MB/s')   100 MB/s      99 MB/s     inf MB/s
+    #:   (100, None, 0.02, 'MB/s')    100 MB/s      -inf MB/s   102 MB/s
+    #:   (-100, -0.01, 0.02, 'C')     -100 C        -101 C      -98 C
+    #:   (-100, -0.01, None, 'C')     -100 C        -101 C      inf C
+    #:   (-100, None, 0.02, 'C')      -100 C        -inf C      -98 C
+    #:   ===========================  ============  ==========  ===========
+    #:
+    #: More information in the `assert_reference function
+    #: <deferrable_functions_reference.html#reframe.utility.sanity.assert_reference>`__.
```

Comment:
I don't know how to shorten this line, so that pep8 doesn't complaint. `#noqa` comment doesn't work, because it will appear in the docs. I could omit it completely. I don't know if `assert_reference` gives much more details.